doc/source/setup-jupyterhub.rst

```diff
@@ -74,7 +74,7 @@ Install JupyterHub
       ...Skip local chart repository
       ...Successfully got an update from the "stable" chart repository
       ...Successfully got an update from the "jupyterhub" chart repository
-      Update Complete. ⎈ Happy Helming!⎈ 
+      Update Complete. ⎈ Happy Helming!⎈
```

Comment:
this is because atom automatically removes trailing whitespace at the end of lines :-P